ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -204,6 +204,8 @@ data ThreadNetworkArgs m blk = ThreadNetworkArgs
   , tnaRestarts     :: NodeRestarts
   , tnaTopology     :: NodeTopology
   , tnaTxGenExtra   :: TxGenExtra blk
+  , tnaVersion      :: NodeToNodeVersion
+  , tnaBlockVersion :: BlockNodeToNodeVersion blk
```

Comment:
Let it vary per node. Make this part of `TestNodeInitialization`?

Reply:
It that's not so straightforward, create an issue for it and add it to the list of issues on the EBB decontamination plan.

Reply:
See #2359.